input/http.js

```diff
@@ -11,8 +13,11 @@ function create ({name = 'HTTP endpoint', endpoint, parse}) {
         let messages = Array.isArray(req.body) ? req.body : [req.body]
         messages.forEach(message => {
           try {
-            const log = parse(req.body)
-            pipeline.success(log)
+            if (parse) {
```

Comment:
Why don't you just set a default value for `parse` to `fromJS`?

Reply:
that could work there, but for the text based inputs, that is ```fromJS(JSON.parse(msg))``` so less simple.